test/appium/support/api/network_api.py

```diff
@@ -1,73 +1,66 @@
 import logging
+import time
+from decimal import Decimal
+from json import JSONDecodeError
+from os import environ
 from typing import List
 
 import pytest
 import requests
-import time
-from json import JSONDecodeError
-from decimal import Decimal
-from os import environ
+from selenium.common import TimeoutException
+
 import tests
 
 
-class NetworkApi(object):
+class NetworkApi:
     def __init__(self):
-        self.network_url = 'http://api-goerli.etherscan.io/api?'
-        self.headers = {
-        'User-Agent':"Mozilla\\5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit\\537.36 (KHTML, like Gecko) Chrome\\7"
-                     "7.0.3865.90 Safari\\537.36", }
-        self.chat_bot_url = 'http://offsite.chat:8099'
+        self.network_url = 'http://api-sepolia.etherscan.io/api'
         self.api_key = environ.get('ETHERSCAN_API_KEY')
 
     def log(self, text: str):
         tests.test_suite_data.current_test.testruns[-1].steps.append(text)
         logging.info(text)
 
-    def send_etherscan_request(self, method, extracted_param: str):
-        for attempt in range(3):
-            try:
-                response = requests.request('GET', url=method, headers=self.headers).json()
-                if response:
-                    return response[extracted_param]
-            except TypeError as e:
-                self.log("Check response from etherscan API. Returned values do not match expected. %s" % str(e))
-            except JSONDecodeError as e:
-                self.log("No valid JSON response from Etherscan: %s " % str(e))
-                pass
-            time.sleep(30)
```

Comment:
I do not exactly remember why we had it here, it might be due to some limitation on etherscan API. Are you sure that we need to remove it

Reply:
We can add it anytime. For now I decided not to use it because of the time limits. Current api key limits should be enough for current amount of requests